packages/base/src/index.ts

```diff
@@ -12,6 +12,7 @@ governing permissions and limitations under the License.
 
 export * from './Base.js';
 export * from './sizedMixin.js';
+export * from './streaming-listener.js';
```

Comment:
I want to change this part, just a little bit. Let's take a look together and get it cleaned up before we merge and go ahead with an alpha release here.